2/src/styles/_doubleContainer.scss

```diff
@@ -0,0 +1,215 @@
+.double-container {
+  background: linear-gradient(to right top, #b0adad 10%, white 60%);
+}
+
+.problems {
+  font-family: Aclonica, sans-serif;
+
+  h2 {
+    padding-top: 120px;
+  }
+
+  .sl1-item {
+    display: grid !important;
```

Comment:
why do you need `!important` here?

Reply:
I need here important because slick-slider add display:inline-block in HTML and the only way to overwrite it is to add !important